src/test/java/com/example/document/controller/DocumentControllerTest.java

```diff
@@ -180,22 +167,14 @@ void viewDocumentEntry() throws Exception {
         assertThat(actualDocumentEntryDTO)
                 .usingRecursiveComparison()
                 .ignoringFields("id")
-                .isEqualTo(documentEntryDTO);
+                .isEqualTo(documentEntryDTOS.get(0));
     }
 
+
     @Test
     void viewMultipleDocumentEntries() throws Exception {
         List<DocumentEntryDTO> documentEntryDTOS = new ArrayList<>();
-        for (long i = 0; i < 5; i++) {
-            DocumentEntryDTO documentEntryDTO = DocumentEntryDTO.builder()
-                    .uploadDate(LocalDate.of(2022, 10, 13))
-                    .effectiveDate(LocalDate.of(2022 + (int) i, 9, 30))
-                    .employeeId(1L)
-                    .benefitId(2L)
-                    .amount(BigDecimal.valueOf(300))
-                    .build();
-            documentEntryDTOS.add(documentEntryDTO);
-        }
+        IntStream.range(0, 5).forEach(i -> documentEntryDTOS.add(createDocumentEntryDTO(i)));
```

Comment:
Let's do it in a bit different manner
Instead of using `foreach` we can use `mapToObj` where we will create document entry with each number and then we will collect them to one list